dlnvalidation.py

```diff
@@ -12,7 +12,7 @@
     'CT': r'^[0-9]{9}$',
     'DE': r'^[0-9]{1,7}$',
     'DC': r'^[0-9]{7}$|^[0-9]{9}$',
-    'FL': r'^[a-z][0-9]{1,12}$',
+    'FL': r'^\A[a-zA-Z]{1}\d{12}\z$',
```

Comment:
You don't need the `^` and `\A` at the same time. Same with `$` and `\z`
```suggestion
    'FL': r'^[a-zA-Z]{1}\d{12}$',
```